sympy/sets/sets.py

```diff
@@ -1628,6 +1628,10 @@ class FiniteSet(Set, EvalfMixin):
     >>> 3 in FiniteSet(1, 2, 3, 4)
     True
 
+    >>> members = [1, 2, 3, 4]
+    >>> FiniteSet(*members)
```

Comment:
Hmm, this is an exact equivalent of the L1626.  Should we add doctring for every Python's feature.  I don't think so, we should assume that people learn Python first and then - SymPy.